pkg/sql/testdata/iris_sql.go

```diff
@@ -445,15 +445,15 @@ INSERT INTO gomaxcompute_driver_w7u.sqlflow_test_iris_train VALUES
 (5.6,2.9,3.6,1.3,1),
 (4.8,3.1,1.6,0.2,0);
 
-DROP TABLE IF EXISTS gomaxcompute_driver_w7u.sqlflow_test_iris_test;
-CREATE TABLE gomaxcompute_driver_w7u.sqlflow_test_iris_test(
+DROP TABLE IF EXISTS %s.sqlflow_test_iris_test;
+CREATE TABLE %s.sqlflow_test_iris_test(
        sepal_length DOUBLE,
        sepal_width  DOUBLE,
        petal_length DOUBLE,
        petal_width  DOUBLE,
        class BIGINT);
 
-INSERT INTO gomaxcompute_driver_w7u.sqlflow_test_iris_test VALUES
+INSERT INTO %s.sqlflow_test_iris_test VALUES
```

Comment:
Use `%[1]s` instead of `%s`?  So we need only pass  `caseDB` once in `main_test.go`